### ISSUE TITLE
Macros to switch What the Window Shoes does not work anymore

### ISSUE THREAD

**Issue by bawe** [Tercioo/Details-Damage-Meter#485]
**Game and Details! version**

DETAILS! VERSION: WD 10730 RELEASE 10.0.7 

**Describe the bug**
Macro with:
```
/script Details:GetWindow(1):SetDisplay( DETAILS_SEGMENTID_CURRENT,6, 5 )
/script Details:GetWindow(2):SetDisplay( DETAILS_SEGMENTID_OVERALL,6, 5 )

```

**Comment by Flamanis** [comment]
There have been reports of this, but they shouldn't error at all. The functions have not changed arguments at all.

**Comment by Flamanis** [comment]
Although it appears when looking at your code, there is no attribute 6. That code, no matter what it actually is, will set it to Damage Done.

**Comment by bawe** [comment]
Ok, you are right, the params are not correct, but had worked until the last days. 

Both windows should display "Damage Done" - CURRENT and OVERALL:
```
/script Details:GetWindow(1):SetDisplay( DETAILS_SEGMENTID_CURRENT,1, 1 )
/script Details:GetWindow(2):SetDisplay( DETAILS_SEGMENTID_OVERALL,1, 1 )
```
But also don't work.

thanks

**Comment by Tercioo** [comment]
Try:
/script Details:SetDisplay(Details:GetWindow(1), DETAILS_SEGMENTID_CURRENT,1, 1 )
/script Details:SetDisplay(Details:GetWindow(2), DETAILS_SEGMENTID_OVERALL,1, 1 )

**Comment by bawe** [comment]
unfortunately no change. no error message/output

**Comment by Tercioo** [comment]
right, so, could you update to latest alpha version and try again?

--set the window 1 to show current segment > damage done
/script Details:GetWindow(1):SetDisplay( DETAILS_SEGMENTID_CURRENT,1, 1 )
--set the window 2 to show overall data > damage done
/script Details:GetWindow(2):SetDisplay( DETAILS_SEGMENTID_OVERALL,1, 1 )

**Comment by bawe** [comment]
with version WD 10730 ALPHA 1 10.0.7 it works
<3

**Comment by bawe** [comment]
works again with release version
thanks!
